languages/common-lisp/reference/types/cons.md

```diff
@@ -0,0 +1,10 @@
+# Cons Cells
+
+A [cons cell][cons] (also known as a dotted pair due its printed form) is a data structure consisting of a pair of two objects. Cons cells are created with the function `cons` and the items of a cons cell are accessed with `car` and `cdr`.
```

Comment:
Looks excellent! We could mention `first` and `rest`? Not too fussed either way!

Reply:
I vassilated on this one. `first` and `rest` to me feel like things that are specifically for lists.

But i will add them parenthetically.

Reply:
Hmm, I somewhat agree actually. They probably don't need to be there. Up to you!